### PR TITLE
Support emojis in new UI

### LINKED ISSUE #2802: Support emojis in new UI
When running the default feature extraction there are rules like [gmail_accounts](https://github.com/google/timesketch/blob/master/data/features.yaml#L74) which are adding an emoji to the event. However, this seems not to work in the new UI yet.

**Old UI:** 
image

**New UI:**
image

### PR DESCRIPTION
**Pull request by NightAcrobat777**
**IMPORTANT: All Pull Requests should be connected to an issue if you don't
have an issue, please start by creating an issue and link it to the PR.**

Please provide enough information so that others can review your pull request:

We changed the index from === 0 to === 3 since the headers in the new UI are different from the old UI in line 362 of the EventList.vue on the frontend ng file. We also removed the redundant code that renders the emoji.

@raylandho and @aduarte-vasquez, have also aided in this pull request as a team.

+ What existing problem does this PR solve?
+ What new feature is being introduced with this PR?
+ Overview of changes to existing functions if required.

**Checks**

- [x] All tests succeed.
- [x] Unit tests added. (not needed)
- [x] e2e tests added. (not needed)
- [x] Documentation updated. (not needed)

**Closing issues**

closes #2802

### CODE REVIEW

File: .zprofile
Comment (jkppr) [inline .zprofile:1]: Please keep your PR clean. This file is not needed for this PR, please delete!

File: get-pip.py
Comment (jkppr) [inline get-pip.py:1]: Please keep your PR clean. This file is not needed for this PR, please delete!

File: test_tools/etc/.DS_Store
Comment (jkppr) [inline test_tools/etc/.DS_Store:1]: Please keep your PR clean. This file is not needed for this PR, please delete!

File: timesketch/frontend-ng/yarn.lock
Comment (jkppr) [inline timesketch/frontend-ng/yarn.lock:1]: Please keep your PR clean. This file is not needed for this PR, please delete!

File: yarn.lock
Comment (jkppr) [inline yarn.lock:1]: Please keep your PR clean. This file is not needed for this PR, please delete!

**Review by jkppr — CHANGES_REQUESTED** [review]
Thanks for the fix. Please ensure to submit a clean PR that only contains the files needed for the change. It is also recommended to submit changes from a separate branch of your fork and not from the master branch.

Thanks.

**Comment by jkppr** [comment]
I have tested the fix to bring back emojis in the new UI and it works well, thanks! 

Somehow I cannot delete the `.DS_store` file from the PR. Please remove this file from the PR and we are ready to merge the fix and include it in the next release!

**Comment by NightAcrobat777** [comment]
> I have tested the fix to bring back emojis in the new UI and it works well, thanks!
> 
> Somehow I cannot delete the `.DS_store` file from the PR. Please remove this file from the PR and we are ready to merge the fix and include it in the next release!

I just made the change.

**Pull request merged**